ts/components/services/ServicesSectionsList.tsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * A component to render a list of services organized in sections, one for each organization.
+ */
+import { Option } from "fp-ts/lib/Option";
+import I18n from "i18n-js";
+import { Button, Text, View } from "native-base";
+import React, { ComponentProps } from "react";
+import { Image, NativeScrollEvent, NativeSyntheticEvent } from "react-native";
+import { StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+import IconFont from "../ui/IconFont";
+import ServiceList from "./ServiceList";
+import ServiceSectionListComponent from "./ServiceSectionListComponent";
+
+type AnimatedProps = {
```

Comment:
I guess these properties too are useless.
Could you check?

Reply:
I checked. They are used to animate the header at the content scroll (without them the header is not hidden)